react/src/components/pagination/PaginationContent.jsx

```diff
@@ -0,0 +1,33 @@
+// Externals - Others
+import _ from 'lodash';
+import React from 'react';
+import { ListGroup, ListGroupItem } from 'reactstrap';
+
+
+const PaginationContent = (props) => {
+  const {
+    EntityComponent,
+    entities,
+    className,
+  } = props;
+
+  return (
+    <div className={`${className}`}>
+      <ListGroup>
+        {_.map(entities, (entity, index) => (
```

Comment:
They behave slightly differently, not that it makes difference in this case. Also lodash's methos are usually faster and is more consistent with rest of my code

http://jsperf.com/array-map-vs-lo-dash-map